### DISCUSSION TITLE
In-game website leaderboard for osu!mania beatmaps shows all scores as 0

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#18823] [Q&A] [answered]
This occurs with every osu!mania beatmap that has a leaderboard. The actual in-game leaderboard that isn't tied to the website functions as normal.

![osu_2022-06-23_12-53-05](https://user-images.githubusercontent.com/74088315/175363237-7e3d0141-da7f-499d-b8fb-d9f51a94928d.png)

**Comment by bdach** [comment, marked as answer]
#18824
